parameter_files/ProjectParameters_PA2024CH.yml

```diff
@@ -0,0 +1,66 @@
+default:
+
+    paths:
+        data_location_ext: ../pacta-data/2023Q4/
+
+    reporting:
+        project_report_name: PA2024CH
+        display_currency: CHF
+        currency_exchange_value: 1.190405
+
+    parameters:
+        start_year: 2023
+        horizon_year: 5
+        select_scenario: WEO2023_NZE_2050
+        scenario_other: GECO2023_1.5C-Unif
+        portfolio_allocation_method: portfolio_weight
+        scenario_geography: Global
+
+    methodology:
+        has_map: TRUE
+        has_revenue: FALSE
+        has_credit: FALSE
+        inc_emissionfactors: TRUE
+
+    sectors:
+        tech_roadmap_sectors:
+            - Power
+            - Automotive
+            - Oil&Gas
+            - Coal
+        pacta_sectors_not_analysed:
+            - Steel
+            - Aviation
+            - Cement
+        green_techs:
+            - RenewablesCap
+            - HydroCap
+            - NuclearCap
+            - Hybrid
+            - Electric
+            - FuelCell
+            - Electric Arc Furnace
+        alignment_techs:
+            - RenewablesCap
+            - CoalCap
+            - Coal
+            - Oil
+            - Gas
+            - Electric
+            - ICE
+
+    scenario_sources_list:
+        - GECO2023
+        - ISF2023
+        - WEO2023
+
+    scenario_geography_list:
+        - Global
+        - GlobalAggregate
+        - NonOECD
+        - OECD
```

Comment:
`OECD` and `nonOECD` is not included because there is no data specific to these regions. 
There is data for, e.g. `EU27`, and further country level pathways (e.g. `Australia`) but these won't get picked up. 

See my comment to that effect here:
https://dev.azure.com/RMI-PACTA/2DegreesInvesting/_workitems/edit/9355#30498012

Reply:
gonna leave `NonOECD` and `OECD` in aspirationally, because we want them in but the data has to provide it